/**:
  ros__parameters:
    # Storage parameters:
    # map_path is concatened with pose_graph_file_name to create this file 
    # TODO: should I find a way to do this with a relative path?
    map_path: /home/romantwice/Projects/MISTLab/Swarm-SLAM/src/cslam_visualization/_map
    pose_graph_file_name: pose_graph.json
    enable_map_reading: false 
    enable_map_storage: false
    is_robot: false
    max_nb_robots: 2